QUARKUS-1405.md

```diff
@@ -0,0 +1,24 @@
+# Quarkus-1405 - Mutiny 1.x improvements
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1405
+
+This test plan cover Mutiny 1.x improvements, more in detail the concept of mutiny [context](https://smallrye.io/smallrye-mutiny/guides/context-passing).
+
+## Scope of the testing
+
+Mutiny contexts shall be primarily used to share transient data used for networked I/O processing such as correlation identifiers, tokens, etc. They should not be used as general-purpose data structures that are frequently updated and that hold large amounts of data.
+
+Test development will focus on cover the notion of mutiny `context` and how to pass this metadata values between several components.
+
+### Impact on testsuites and testing automation:
+
+Reusing some existing scenarios as `Rest-client-reactive` where are involved two or more components, could be a good approach. In the end, the most important is to be sure that the context is propagated between several components.  
```

Comment:
context is propagated between several components ... what type of data will you propagate?

https://github.com/smallrye/smallrye-mutiny/pull/776/files#diff-0a4ca8cde3da2b020a94dbe2b82e05eec9684e2492c69ac4c356529edea82216R36 uses String

Other types could be checked too, maybe custom objects?

Reply:
in theory, based in the context API, other objects are supported: 

```
public Context put(String key, Object value)
```